src/java.base/share/classes/jdk/internal/foreign/AbstractMemorySegmentImpl.java

```diff
@@ -133,10 +135,13 @@ public AbstractMemorySegmentImpl asSlice(long offset) {
     /**
      * Mismatch over long lengths.
      */
-    public static long vectorizedMismatchLargeForBytes(MemorySessionImpl aSession, MemorySessionImpl bSession,
-                                                        Object a, long aOffset,
-                                                        Object b, long bOffset,
-                                                        long length) {
+    public static long vectorizedMismatchLargeForBytes(MemorySessionImpl aSession,
```

Comment:
So, this is a decision whether to break up one per line, or break up so that we never exceed 100 chars per line. I don't have any strong opinion here, but I note that the JDK code is not consistent about this at all.